lib/cisco_node_utils/node.rb

```diff
@@ -57,7 +57,7 @@ def config_get(feature, property, *args)
       return ref.default_value if ref.default_value? && !ref.getter?
 
       get_args, ref = massage_structured(ref.getter(*args).clone, ref)
-      data = get(command:     ref.get_command,
+      data = get(command:     get_args[:command],
```

Comment:
We need to make sure we run a full minitest and beaker tests with this change just be be on the safe side.